modules/ldsg.js

```diff
@@ -0,0 +1,147 @@
+// @ts-check
+
+const Augur = require("augurbot-ts"),
+  u = require("../utils/utils"),
+  Discord = require("discord.js"),
+  { ButtonStyle } = require("discord.js");
+
+/**
+ * Responds with the number of guild members, and how many are online.
+ * @param {Discord.ChatInputCommandInteraction} interaction The interaction that the user submits.
+ */
+async function slashLdsgMembers(interaction) {
+  try {
+    const ldsg = interaction.client.guilds.cache.get(u.sf.ldsg);
+    if (!ldsg) throw new Error("Couldn't find LDSG");
+    const online = ldsg.members.cache.filter((member) => member?.presence && member.presence.status != "offline");
+    const response = `📈 **Members:**\n${ldsg.memberCount} Members\n${online.size} Online`;
+    await interaction.reply({ content: response });
+  } catch (error) { u.errorHandler(error, interaction); }
+}
+
+const sendOptions = [
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("pa").setEmoji("🗣️").setLabel("Public Affairs").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("log").setEmoji("🚚").setLabel("Logistics").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("ops").setEmoji("🗓️").setLabel("Operations").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("mgmt").setEmoji("⚙️").setLabel("Management").setStyle(ButtonStyle.Primary),
+    new u.Button().setCustomId("icarus").setEmoji("🤖").setLabel("Icarus").setStyle(ButtonStyle.Primary),
```

Comment:
@LDSGOhGo custom IDs should be more unique. maybe more like `suggestionPA` or `suggestionLog`